behat/symfony2-extension/2.1/features/bootstrap/bootstrap.php

```diff
@@ -0,0 +1,10 @@
+<?php
+
+use Symfony\Component\Dotenv\Dotenv;
+
+require __DIR__.'/../../vendor/autoload.php';
```

Comment:
this line is useless. Behat already loads the composer autoloading file much earlier than the loading of this file by the extension